notebooks/vertex_matching_engine/solutions/matching_engine_query.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>Once the dependency is in the <code>requirments.txt</code> maybe just have a markdown cell mentioning the user should possibly install that library. A word about what this lib is about would be useful.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>